third_party/pcsc-lite/naclport/build_configuration/Makefile

```diff
@@ -0,0 +1,56 @@
+# Copyright 2022 Google Inc.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#
+# 1. Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+# 2. Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+# 3. The name of the author may not be used to endorse or promote products
+#    derived from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE AUTHOR ``AS IS'' AND ANY EXPRESS OR
+# IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+# OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
+# IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT,
+# INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT
+# NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+# DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+# THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
+# THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+# This makefile runs the PC/SC-Lite's "bootstrap" and "configure" scripts, in
+# order to prepare autogenerated headers that are used by our webports of
+# PS/SC-Lite and CCID.
+#
+# Note that we don't take the "CONFIG"/"TOOLCHAIN"/etc. environment variables
+# into account, since the steps performed here don't depend on these and should
+# only be executed once.
+
+# Define the default target (executed when running "make" without any extra
+# arguments). Redirect the actual work to the "build.stamp" helper.
+.PHONY: all
+all: build.stamp
+
+# The helper "build.stamp" target performs the needed commands whenever
+# something in the ../../src folder (where PC/SC-Lite's original sources are) is
+# changed.
+#
+# Before running the build, we clean old build artifacts. On completion, the
+# target creates build.stamp that allows Make to track file timestamps and
+# detect changes.
+build.stamp: $(shell find ../../src)
+	@rm -f build.stamp
+	@git status --ignored ../../src | xargs rm -rf
+	cd ../../src && ./bootstrap && ./configure
+	@touch build.stamp
```

Comment:
I assume this instruction is unaffected by the `cd` in the line above?

Reply:
Correct, the current directory is reset to the original one in every line in the recipe (IIUC, because technically each line runs in a subshell).